### DISCUSSION TITLE
Packaging

### DISCUSSION THREAD

**Discussion by ShahwaizRaza** [TomSchimansky/CustomTkinter#1506] [General]
I have created a program for Windows and converted it into an EXE file. now I want to create a Mac app. What can I do? what will the process be?

**Comment by TomSchimansky** [comment]
You have to use py2app.

**Reply by alfmelchor** [reply]
I've been using py2app but I keep getting an error saying that the JSON file for the CTK theme is not found. I've followed the instructions on the wiki but still run into issues.

**Reply by ShahwaizRaza** [reply]
I do use Py2App, but I'm not sure how to do it correctly. Can you add the right procedure to your documentation as you do for Windows using pyinstaller?
**[Your documentation!](https://customtkinter.tomschimansky.com/documentation/packaging)**

it will be very helpful for us
Thanks ☺️

> You have to use py2app.
